hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -92,7 +94,8 @@
 		static final int TOKENS = 5;
 		static final int NUM_080_CHILDREN = 6;
 		static final int TOKEN_ASSOCIATIONS = 6;
-		static final int NUM_090_CHILDREN = 7;
+		static final int SPECIAL_FILE_SYSTEM = 7;
```

Comment:
I am still not sure why the update jar files need to be part of state. Are we making things too complicated?